pkg/interactive-installer/README.md

```diff
@@ -0,0 +1,31 @@
+# EVE-OS interactive installer
+
+## Description
+
+The EVE-OS interactive installer provides a user friendly environment for the installation of EVE-OS. It allows users to configure the paramenters of the installation using a TUI (text-based UI). The primary goal of this installer mode is to facilitate the installation of EVE-OS on-edge devices while providing flexibility, and automation.
```

Comment:
These two lines are vTheseery similar but not identical. Please delete one of the.

Note thaI can't comment on this PR since it is too lfor github to display the diffs.

Hence the jibberish in this commen